experimental/traffic-portal/.eslintrc.json

```diff
@@ -237,7 +237,12 @@
 						"enableFixer": false
 					}
 				],
-				"@typescript-eslint/unbound-method": "error",
+				"@typescript-eslint/unbound-method": [
+					"error",
+					{
+						"ignoreStatic": true
+					}
+				],
```

Comment:
We can't add this exception in general, because [the `this` variable in "static" methods is bound to the class itself](https://www.typescriptlang.org/play?#code/MYGwhgzhAEAqCmEAu0DeAoaXoAcCuARiAJbDTJhKnQCOeiSAFAJQBc0AbgPbEAmambEOBcAdhC4h4AOhBcA5oyQALYhGYBuQVgC+6PegTJpdBiyA)

If a method is _truly_ static in that it doesn't reference `this` at all, then it's safe to use unbound, but unlike, say, Python, TypeScript/JavaScript doesn't distinguish between "static" methods and "class" methods, so in general this is still unsafe.

Reply:
removed